src/tools/rebuild_mondb.cc

```diff
@@ -215,7 +215,10 @@ int update_osdmap(ObjectStore& fs, OSDSuperblock& sb, MonitorDBStore& ms)
 
   // trim stale maps
   unsigned ntrimmed = 0;
-  for (auto e = first_committed; e < sb.oldest_map; e++) {
```

Comment:
@ktdreyer, yes. the code change spilled out into the doc change in the original commit. let me fix it.